### PR TITLE
Jk 19 create login page

### LINKED ISSUE #19: feat(f1):create login page
1. generate new component "login component"
2. create input fields for username and password
3. create a login button

### PR DESCRIPTION
**Pull request by swz22**
## Changes
_List Changes Introduced by this PR_
1. Create Login Component
2. Updated Routing Module
3. Changed navbar links to buttons
Closes #19

### CODE REVIEW

**Review by svoggu — COMMENTED** [review]
Code looks good and its running correctly. But i see the login credentials entirely different from what we had in our prototype.Is it that i missed something or not sure. But because signup page have username, password we should use those things for login. Game pin is for players, they need not login or register. 
The page you created will be perfect for Play or Join Game page or Lobby page whatever we call it.

**Comment by Aman-codde** [comment]
> Code looks good and its running correctly. But i see the login credentials entirely different from what we had in our prototype.Is it that i missed something or not sure. But because signup page have username, password we should use those things for login. Game pin is for players, they need not login or register.
> The page you created will be perfect for Play or Join Game page or Lobby page whatever we call it.

Exactly! this is going to be normal login page with **username** and **password** as input fields, as described in the **issue**,  we will create another issue for play/join-game

File: src/client/src/app/components/login/login.component.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="main-div">
+    <div class="sub-main-div">
+        <div class="form-div">
+            <form>
+                <input type="number" placeholder="Enter Game Pin" formControlName="gamepin" />
```
Comment (Aman-codde) [inline src/client/src/app/components/login/login.component.html]: this is a basic login page, so there is going to be **username** and **password** input field, and a **login** button.
Also, 
1. formControlName should be **loginForm**
2. login button should have [disabled]="loginForm.invalid" for making button disabled until the input fields are empty

File: src/client/src/app/components/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+
```
Comment (Aman-codde) [inline src/client/src/app/components/login/login.component.ts]: Please add loginForm as FormGroup and Formbuilder in the constructor to describe the username and password are required

**Review by Aman-codde — CHANGES_REQUESTED** [review]
Good job on creating the PR, but there are some major changes that I mentioned inside the files, once done with changes, it will be good to go after that

**Review by Aman-codde — APPROVED** [review]

**Pull request merged**